### DISCUSSION TITLE
Paginas geniales para aprender JavaScript

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#338] [Ideas]
Hola explorers
les comparto unas paginas para que lleven sus habilidades de JS al infinito y más allá
[#JavaScript30](https://javascript30.com/)
[Lenguaje JS](https://lenguajejs.com/)
[JetBrains Academy: FrontEnd Developer](https://hyperskill.org/tracks/5)
[Freecode camp JS](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/)
[Codecademy: Learn JS](https://www.codecademy.com/learn/introduction-to-javascript)
[YouTube JavaScript desde cero con Lucas Dalto](https://www.youtube.com/watch?v=z95mZVUcJ-E)
[Udemy: HTML CSS JS desde cero](https://www.udemy.com/course/aprende-javascript-es9-html-css3-y-nodejs-desde-cero/)
En mi opinión el más completo es el de jetbrains academy, tienen lecciones en temas bien seccionados y con ejercicios por cada tema... aunque hay 2 "peros"
1 Tooooodo esta en ingles
2 Para que te evaluen los proyectos alrededor de los cuales iras practicando necesitas una licencia de Webstorm (gratis si tienes un correon institucional) ya que el curso se conecta directamente con el IDE

**Comment by AngelAlfaro92** [comment]
Nuestro corrreo de Inovaccion cuenta como correo institucional no??

**Reply by carlos-gmx3** [reply]
No se, sería cuestión de que hagas la prueba y nos cuentes.
Yo usé el correo que me dio la universidad en la que estoy estudiando
